src/components/bcros/filing/addStaffFiling/Index.vue

```diff
@@ -10,6 +10,7 @@ interface MenuActionItem extends DropdownItem {
 
 const filings = useBcrosFilings()
 const business = useBcrosBusiness()
+const { getStoredFlag } = useBcrosLaunchdarkly()
```

Comment:
I was wondering about your implementation of the LD client so I looked at your code. Why don't you have streaming enabled? That would get you real-time flag updates, I think...?

Reply:
I am not sure how that works, but I like the idea, will look into it when I get time for it. Thanks !